src/main/java/com/labsynch/labseer/utils/SecurityUtils.java

```diff
@@ -66,7 +67,7 @@ public void updateAuthorInfo(Authentication authentication) {
 
 	}
 
-	private Author createAuthor(Authentication authentication) {
+	private Author creacreateAuthorteAuthor(Authentication authentication) {
```

Comment:
copy/paste issue

Reply:
Thanks, somehow did that after building and deploying/testing